packages/oui-message/src/message.controller.js

```diff
@@ -8,14 +8,13 @@ export default class {
     }
 
     $onInit () {
+        // Guidelines default value for dismissable attribute
+        addDefaultParameter(this, "dismissable", this.type === "info" || this.type === "success");
```

Comment:
Could have an array of types that are dissimisable by default

Reply:
What's your proposition ?

Reply:
`message.constants.js`

```js
angular
	.module("moduleName")
	.constant("MESSAGE_CONSTANTS", () => {
		TYPES: {
			info: {
				dismissableByDefault: true
			},
			...
		}
})
```

`message.controller.js`

```js
addDefaultParameter(this, "dismissable", this.MESSAGE_CONSTANTS.TYPES[this.type].isDismissableByDefault
```

The constants could contain more data about types and would make it easier to add new types, if the need ever rise.

I would understand if you considered it over-engineering 😄